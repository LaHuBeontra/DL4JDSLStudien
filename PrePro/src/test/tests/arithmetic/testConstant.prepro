const test = 42.53;
export test;
