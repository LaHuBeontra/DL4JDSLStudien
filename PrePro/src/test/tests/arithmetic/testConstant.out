DefaultPreProDataSet{
dataSetEntries=
[],
dataSet=
null

constants={test=42.53}

calculated_variables=
test=42.53
}